src/SimpleInjector.Tests.Unit/InstanceProducerTests.cs

```diff
@@ -110,12 +110,16 @@ public void VisualizeObjectGraph_NestedObjectGraphWithMultipleDecorators_BuildsT
         {
             // Arrange
             string expectedObjectGraph =
-@"PluginDecorator(
-    PluginDecoratorWithDependencyOfType<FakeTimeProvider>(
-        FakeTimeProvider(),
-        PluginDecorator<Int32>(
-            PluginWithDependencyOfType<RealTimeProvider>(
-                RealTimeProvider()))))";
+@"PluginDecorator( // Transient
+    PluginDecoratorWithDependencyOfType<FakeTimeProvider>( // Transient
+        FakeTimeProvider(), // Transient
+        PluginDecorator<Int32>( // Transient
+            PluginWithDependencyOfType<RealTimeProvider>( // Transient
+                RealTimeProvider() // Transient
+            )
```

Comment:
With the risk of asking something that might be extermely difficult to implement (haven't looked at this myself very closely), could you prevent having the closing brackets on new lines? So is it possible to achieve the following:

```c#
`PluginDecorator( // Transient
    PluginDecoratorWithDependencyOfType<FakeTimeProvider>( // Transient
        FakeTimeProvider(), // Transient
        PluginDecorator<Int32>( // Transient
            PluginWithDependencyOfType<RealTimeProvider>( // Transient
                RealTimeProvider())))) // Transient  <- {note the closing brackets here)
``

Reply:
I'll have a look at it tomorrow :)